e2e/playwright/code-pane-and-errors.spec.ts

```diff
@@ -65,6 +65,8 @@ const extrude001 = extrude(5, sketch001)`
   test('Opening and closing the code pane will consistently show error diagnostics', async ({
     page,
   }) => {
+    await page.goto('http://localhost:3000')
```

Comment:
I was trying things at midnight last night 😆 feel free to delete lol

Reply:
Aw dang sorry man!

Reply:
That had no impact on the test so I left it for now, it's very easy to clean up on another pass, we're talking like an extra 400ms on a test run.